### PR TITLE
Fixes for Crystal_SMD_0603_4Pads.kicad_mod

### PR DESCRIPTION
**Pull request by SchrodingersGat**
- Added courtyard
- Fixed pad numbering to match common crystal pinout convention
- Added pin-1 marking

crystal

### CODE REVIEW

File: Crystal_FOX-FE_SMD.kicad_mod [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
   (fp_circle (center 0 0) (end 0.50038 0) (layer F.Adhes) (width 0.381))
   (fp_circle (center 0 0) (end 0.14986 -0.0508) (layer F.Adhes) (width 0.381))
-  (fp_line (start -3.74904 2.49936) (end -3.74904 1.69926) (layer F.SilkS) (width 0.15))
-  (fp_line (start -3.74904 -2.49936) (end -3.74904 -1.69926) (layer F.SilkS) (width 0.15))
-  (fp_line (start 3.74904 -2.49936) (end 3.74904 -1.69926) (layer F.SilkS) (width 0.15))
-  (fp_line (start 3.74904 2.49936) (end 3.74904 1.69926) (layer F.SilkS) (width 0.15))
-  (fp_line (start -3.74904 2.49936) (end 3.74904 2.49936) (layer F.SilkS) (width 0.15))
-  (fp_line (start 3.74904 -2.49936) (end -3.74904 -2.49936) (layer F.SilkS) (width 0.15))
+  (fp_line (start -3.8 2.6) (end -3.8 1.7) (layer F.SilkS) (width 0.15))
+  (fp_line (start -3.8 -2.6) (end -3.8 -1.69926) (layer F.SilkS) (width 0.15))
```
Comment (pointhi) [inline Crystal_FOX-FE_SMD.kicad_mod]: you should move the end point a little bit, to be the same as all the other ones

**Comment by SchrodingersGat** [comment]
@pointhi I've fixed the line position as requested. Thanks

**Comment by pointhi** [comment]
```
checking module: HC-18UH
  Violating Rule 10.1
    Footprint name must match its filename. (.kicad_mod files).
  Violating Rule 10.4
    Value is filled with footprint name and is placed on the fabrication layer.
checking module: HC-18UV
  Violating Rule 10.1
    Footprint name must match its filename. (.kicad_mod files).
  Violating Rule 10.4
    Value is filled with footprint name and is placed on the fabrication layer.
checking module: HC-33UH
  Violating Rule 10.1
    Footprint name must match its filename. (.kicad_mod files).
  Violating Rule 10.4
    Value is filled with footprint name and is placed on the fabrication layer.
checking module: HC-33UV
  Violating Rule 10.1
    Footprint name must match its filename. (.kicad_mod files).
  Violating Rule 10.4
    Value is filled with footprint name and is placed on the fabrication layer.
checking module: HC-49V
  Violating Rule 10.1
    Footprint name must match its filename. (.kicad_mod files).
  Violating Rule 10.4
    Value is filled with footprint name and is placed on the fabrication layer.
```

(That are only the errors which are invented through renaming). Please also note the related 3D-Models

**Comment by SchrodingersGat** [comment]
I've made a lot more improvements here

**Comment by pointhi** [comment]
```
checking module: Crystal_HC-18UH
  Violating Rule 10.7
    3D Shape ".wrl" files are named the same as their footprint and are placed in a folder named the
    same as the footprint library replacing the ".pretty" with ".3dshapes".
checking module: Crystal_HC-18UV
  Violating Rule 10.7
    3D Shape ".wrl" files are named the same as their footprint and are placed in a folder named the
    same as the footprint library replacing the ".pretty" with ".3dshapes".
checking module: Crystal_HC-33UH
  Violating Rule 10.7
    3D Shape ".wrl" files are named the same as their footprint and are placed in a folder named the
    same as the footprint library replacing the ".pretty" with ".3dshapes".
  Violating Rule 6.6
    Courtyard line has a width 0.05mm. This line is placed so that its clearance is measured from its
    center to the edges of pads and body, and its position is rounded on a grid of 0.05mm.
    No courtyard line found in the module
checking module: Crystal_HC-33UV
  Violating Rule 10.7
    3D Shape ".wrl" files are named the same as their footprint and are placed in a folder named the
    same as the footprint library replacing the ".pretty" with ".3dshapes".
checking module: HC-49V
  Violating Rule 10.1
    Footprint name must match its filename. (.kicad_mod files).
  Violating Rule 10.4
    Value is filled with footprint name and is placed on the fabrication layer.
checking module: Crystal_HC49-U_Horizontal_Typ1
  Violating Rule 6.3
    For through-hole components, footprint anchor is set on pad 1.
  Violating Rule 6.6
    Courtyard line has a width 0.05mm. This line is placed so that its clearance is measured from its
    center to the edges of pads and body, and its position is rounded on a grid of 0.05mm.
    No courtyard line found in the module
checking module: Crystal_Watch
  Violating Rule 10.2
    Doc property contains a full description of footprint.
  Violating Rule 6.6
    Courtyard line has a width 0.05mm. This line is placed so that its clearance is measured from its
    center to the edges of pads and body, and its position is rounded on a grid of 0.05mm.
    No courtyard line found in the module
...
```

I think it would be a good idea to remove the `Crystal_` from the footprint name as done for the connectors Library

**Comment by jkriege2** [comment]
should be fixed by this PR: https://github.com/KiCad/Crystals.pretty/pull/19

**Comment by SchrodingersGat** [comment]
Thanks @jkriege2

**Pull request closed without merging**